### PR TITLE
Output GitHub Actions annotations for `brew style`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Homebrew/utils/github/actions.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module GitHub
```
Comment (MikeMcQuaid) [inline Library/Homebrew/utils/github/actions.rb:3]: Feels like this should be in a `Utils` module too based on the name?

File: Library/Homebrew/style.rb [hunk 2]
```diff
@@ -202,33 +209,58 @@ def json_result!(result)
...
+      def for_path(path)
+        @offenses.fetch(Pathname(path), [])
+      end
+
+      def each(*args, &block)
+        @offenses.each(*args, &block)
+      end
+
+      def to_github_annotations
+        return [] unless (workspace = ENV["GITHUB_WORKSPACE"])
```
Comment (MikeMcQuaid) [inline Library/Homebrew/style.rb:235]:
```suggestion
        workspace = ENV["GITHUB_WORKSPACE"]
        return [] if workspace.blank?
```

**Review by MikeMcQuaid — APPROVED** [review]
Very nice! Some suggestions but nothing blocking.

File: Library/Homebrew/utils/github/actions.rb (hunk 1 above)
Comment (reitermarkus) [reply Library/Homebrew/utils/github/actions.rb:3]: Yeah, but the existing `GitHub` module is not.

Comment (MikeMcQuaid) [reply Library/Homebrew/utils/github/actions.rb:3]: I'd consider moving those files around?

Comment (reitermarkus) [reply Library/Homebrew/utils/github/actions.rb:3]: Which ones? Move `GitHub` out of `utils`?

Comment (MikeMcQuaid) [reply Library/Homebrew/utils/github/actions.rb:3]: Yeh.

Comment (reitermarkus) [reply Library/Homebrew/utils/github/actions.rb:3]: I'm not sure it's worth moving at this moment. I think eventually all modules should follow the file structure and also all of them should be contained in the `Homebrew` namespace.

Comment (MikeMcQuaid) [reply Library/Homebrew/utils/github/actions.rb:3]: Cool.

**Pull request merged**